plugins/openai_plugin.py

```diff
@@ -56,25 +106,23 @@ def request_http(self, query: dict, user_id: int, test_end_time: float = 0):
 
         headers = {"Content-Type": "application/json"}
 
-        if "/v1/chat/completions" in self.host:
-            data = {
-                "messages": [
-                    {"role": "user", "content": query["text"]}
-                ],
-                "max_tokens": query["output_tokens"],
-                "temperature": 0.1,
-            }
-        else:
-            data = {
-                "prompt": query["text"],
-                "max_tokens": query["output_tokens"],
-                "min_tokens": query["output_tokens"],
-                "temperature": 1.0,
-                "top_p": 0.9,
-                "seed": 10,
-            }
+        request = {
+            "max_tokens": query["output_tokens"],
+            "min_tokens": query["output_tokens"],
+        }
+
+        if self.api == 'chat':
+            request["messages"] = [
+                { "role": "user", "content": query["text"] }
+            ]
+        else: # self.api == 'legacy'
+            request["prompt"] = query["text"],
```

Comment:
In favor of not changing the behavior the only thing missing would be `top_p`.

Reply:
I think we should remove top_p for now. In a follow-up PR we should make the sampling parameters configurable

Reply:
That's good point. We should do that.